KsApi/extensions/MockGraphQLClient.swift

```diff
@@ -73,6 +88,18 @@ private func producer<T, E>(for property: Result<T, E>?) -> SignalProducer<T, E>
   }
 }
 
+private func producer<T, E>(for property: Result<T, E>?) -> AnyPublisher<T, E> {
+  switch property {
+  case let .success(data):
+    return CurrentValueSubject(data).eraseToAnyPublisher()
+  case .failure:
+    assertionFailure("Need to implement this behavior. I think the Fail() subject is what we want, possibly with a deferred?")
```

Comment:
We'll have to implement this when we want to actually test failing a network request.

Reply:
Nit: maybe add a `TODO` here, so we don't lose track of it? I expect we'll want to to be able to test failing network requests, especially if we want to improve how we handle them, so doing that as part of a proof-of-concept sounds like a good idea.

Reply:
Will do